service/src/main/java/bio/terra/policy/service/region/RegionService.java

```diff
@@ -88,7 +88,8 @@ public HashSet<String> getDataCentersForRegion(String regionName, String platfor
    */
   @Nullable
   public Region getOntology(String regionName, String platform) {
-    Region mappedRegion = regionNameMap.get(regionName);
+    String queryRegion = (regionName == null || regionName.isEmpty()) ? GLOBAL_REGION : regionName;
```

Comment:
can you fix it for `getDataCentersForRegion` too? or is the region for that one supposed to not be nullable? currently getRegionDataCenters allows nullable RegionName in the query param as well.